modules/nw-coredns-egress-firewall.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/configuring-egress-firewall-ovn.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-coredns-egress-firewall_{context}"]
+= Updating subdomains and resolving wildcard domain names
```

Comment:
```suggestion
= Improved DNS resolution and resolving wildcard domain names
```
Updating subdomains doesn't seem to capture the feature correctly. Does this sound better? Wdyt?

Reply:
yes, this does sound better! thank you